astro/src/content/docs/apis/tenants.mdx

```diff
@@ -15,8 +15,8 @@ import StandardPostResponseCodes from 'src/content/docs/apis/_standard-post-resp
 import StandardPutResponseCodes from 'src/content/docs/apis/_standard-put-response-codes.astro';
 import TenantCopyRequestBody from 'src/content/docs/apis/_tenant-copy-request-body.mdx';
 import TenantPasswordValidationRulesResponseBody from 'src/content/docs/apis/_tenant-password-validation-rules-response-body.mdx';
-import TenantRequestBody from 'src/content/docs/apis/_tenant-request-body.mdx';
```

Comment:
I thought we agreed to use absolute URLs for imports all the time? Has that changed?

Reply:
When did we agree to that?

The "absolute" urls (they are actually a path mapping in the tsconfig) were a necessary evil for the docs migration, but Intellij is stupid and doesn't know how to resolve them.

Relative imports do work, and the yellow squigglies go away, so they should be preferred